### PR TITLE
Hide broken stories and fix paginated table example

### PR DESCRIPTION
**Pull request by tiffwong**
## Description

Very rough review of the Storybook entries.
Changes include:
- Hide `EntityDetail` and `NotificationDrawer` stories since they're breaking
- Prevent stories from nesting if it's the only story for the component
- Use `LinodeRow` for the paginated table story instead to prevent the content from wrapping

## How to test

- Run `yarn storybook`
- Check the relevant stories

### CODE REVIEW

File: packages/manager/src/components/Table/Table.stories.mdx [hunk 1]
```diff
@@ -85,139 +76,77 @@ export const linodes = linodeFactory.buildList(10);
...
+                          backups={linode.backups}
+                          displayStatus={capitalize(linode.status)}
+                          region={linode.region}
+                          status={linode.status}
+                          disk={linode.specs.disk}
+                          memory={linode.specs.memory}
+                          mostRecentBackup={linode.backups.last_successful}
+                          tags={linode.tags}
+                          openTagDrawer={() => null}
+                          type={linode.type}
```
Comment (HanaXu) [inline packages/manager/src/components/Table/Table.stories.mdx:127]: The plan is showing up as empty, can we hardcode a value?

image

```suggestion
                          type={{ label: 'Linode 2 GB' }}
```

**Review by HanaXu — APPROVED** [review]
LGTM, left one non-blocking comment

**Review by DevDW — COMMENTED** [review]
since the `Drag and Drop` story has no content at the moment, we should hide it

**Review by bnussman — APPROVED** [review]
Should `packages/manager/src/components/Placeholder/Placeholder.stories.mdx` be removed? It is empty...

Awesome job updating the Paginationed Table story 🚀

**Review by DevDW — APPROVED** [review]
🚢

**Pull request merged**